oh_staff_ui/views.py

```diff
@@ -132,7 +132,7 @@ def show_log(request, line_count: int = 200) -> HttpResponse:
 @login_required
 def upload_file(request: HttpRequest, item_id: int) -> HttpResponse:
     item = ProjectItem.objects.get(pk=item_id)
-    files = MediaFile.objects.filter(item=item)
+    files = MediaFile.objects.filter(item=item).order_by("sequence")
```

Comment:
Adding a sort order is good, thanks - I don't know why we didn't do this originally.  Now that I see this, one improvement needed: files with the same sequence seem to sub-sort randomly, or maybe by an internal id.  See for example http://127.0.0.1:8000/upload_file/2164 (once production data is loaded locally), where sometimes a master is before submaster, and sometimes after.  

There's no perfect solution - I think we can't trust `MediaFile.create_date()` completely, for example.  But happily, `master` -> `submaster` -> `thumbnail`... I did some testing, and `order_by("sequence", "file_type__file_code")` seems to work well... when the data is correct.

Some migrated data, like for item 1420, is bad.  We had conflicts between file names and locations and DLCS file usage / types.  But for new data this sub-ordering should work, and at some point we may do similar data cleanup projects for non-images.